### PR TITLE
Update metadata to use foliage github backend

### PR DESCRIPTION
**Pull request by andreabedini**
This updates everything (metadata and script) to use foliage github backend. This is so that foliage knows the details about the github repo and can use it for other tihngs (e.g. webpages or adding metadata in the cabal files).

I checked this does not cause any extra changes. On my computer only the signatures change (obviously has I don't have the same keys). Most importantly the hashes of the sdists do not change. Please test again yourself :-)

### CODE REVIEW

File: scripts/add-from-github.sh [hunk 1]
```diff
@@ -61,12 +61,15 @@ fi
...
   local TIMESTAMP=$1
-  local URL=$2
-  local SUBDIR=$3
-  local FORCE_VERSION=$4
+  local REPO_URL=$2
+  local REPO_REV=$3
+  local SUBDIR=$4
+  local FORCE_VERSION=$5
+
+  local REPO=${REPO_URL#https://github.com/}
```
Comment (michaelpj) [inline scripts/add-from-github.sh:69]: what happens if this doesn't match?

**Comment by andreabedini** [comment]
Note that the check passes because I am not changing any timestamp. Nevertheless I could have replaced the packages by changing the url. This is something we should improve upon. Perhaps by adding integrity hashes to the urls.

**Review by michaelpj — APPROVED** [review]
Nice!

File: scripts/add-from-github.sh (hunk 1 above)
Comment (andreabedini) [reply scripts/add-from-github.sh:74]: I added a check.

**Pull request merged**